genienlp/models/base.py

```diff
@@ -68,15 +68,6 @@ def load(cls, save_directory: str, *model_args, **kwargs):
         model = cls(args=args, tasks=tasks, vocab_sets=vocab_sets, save_directory=save_directory, *model_args, **kwargs)
         save_dict = torch.load(full_checkpoint_path, map_location=device)
 
-        # HACK
```

Comment:
Why did you remove this? We still use some older models that are trained with `transformers==4.0`, like the paraphraser.

Reply:
This hack caused dimension errors when loading a trained bart model for sequence classification (with the newest version of transformers). I think best solution is to retrain the paraphraser with the new version if it's only a few models. Otherwise we need to update the condition here as to not interfere with models with a classification head on top. But hacks are generally not good and may cause silent errors again down the line. Finding this was causing the error already took some time...